src/components/LogoutButton.tsx

```diff
@@ -0,0 +1,24 @@
+import { useAuth0 } from "@auth0/auth0-react";
+
+
+
+const LogoutButton = () => {
+  const { logout, isAuthenticated } = useAuth0();
+
+
+  if (isAuthenticated) {
+    return (
+      <>
+        <button
+          className="btn btn-primary logoutBtn"
+          onClick={() => logout({ returnTo: window.location.origin })}
```

Comment:
Puste wiersze w pliku. Tą funkcję mimo że krótką przeniósłbym wyżej do zmiennej a tu przekazał tylko referencję.

Reply:
Nie rozumiem co znaczy, że przekazałbyś wyżej, i co to referencja

Reply:
Definiujesz funkcję przed ifem jako zmienną, a w miejsce onClick przekazujesz jedynie jej nazwę, nie komplikując niepotrzebnie JSX.

Czym jest referencja to must-have nie tylko wobec funkcji https://www.bolimnieweb.pl/blog/wartosci-i-referencje